server/etcdserver/api/v2store/store.go

```diff
@@ -781,15 +781,17 @@ func (s *store) Recovery(state []byte) error {
 		return err
 	}
 
-	s.ttlKeyHeap = newTtlKeyHeap()
+	s.ttlKeyHeap = newTTLKeyHeap()
 
 	s.Root.recoverAndclean()
 	return nil
 }
 
+//revive:disable:var-naming
 func (s *store) JsonStats() []byte {
+	//revive:enable:var-naming
```

Comment:
I decided to ignore this warning, as it is part of the v2 API, which I believe will be deprecated for v3.6.